src/main/java/seedu/address/model/ResumeBook.java

```diff
@@ -291,7 +299,14 @@ public void deleteProject(Project key) {
         projects.remove(key);
         for (Item item : resumes) {
             Resume resume = (Resume) item;
-            resume.getProjectIds().remove(Integer.valueOf(id));
+            Resume newresume = new Resume(resume.getName(), resume.getId(), resume.getTags());
+            newresume.setInternshipIds(resume.getInternshipIds());
+            List<Integer> newProjectList = new ArrayList<>();
+            newProjectList.addAll(resume.getProjectIds());
+            newresume.setProjectIds(newProjectList);
+            newresume.setSkillIds(resume.getSkillIds());
+            newresume.getProjectIds().remove(Integer.valueOf(id));
+            setResume(resume, newresume);
```

Comment:
Making it consistent with internship:
```suggestion
            Resume newResume = new Resume(resume.getName(), resume.getId(), resume.getTags());
            List<Integer> newProjectList = new ArrayList<>();
            newProjectList.addAll(resume.getProjectIds());
            
            newResume.setInternshipIds(resume.getInternshipIds());
            newResume.setProjectIds(newProjectList);
            newResume.setSkillIds(resume.getSkillIds());
            newResume.getProjectIds().remove(Integer.valueOf(id));
            
            setResume(resume, newResume);
```